src/edu/csus/ecs/pc2/imports/ccs/ContestSnakeYAMLLoader.java

```diff
@@ -2740,6 +2752,10 @@ public Problem loadCCSProblemFiles(IInternalContest contest, String dataFileBase
 
             ArrayList<SerializedFile> dataFiles = new ArrayList<SerializedFile>();
             ArrayList<SerializedFile> answerFiles = new ArrayList<SerializedFile>();
+            
```

Comment:
Since you wrote a routine to load datafiles from an arbitrary directory, why not use loadDataFiles(problem, dataFiles, answerFiles, dataFileBaseDirectory, loadExternalFile); and fix up loadDataFiles so it can work for both?  I understand there will have to be some small changes to the stuff that sets up the dataFiles and answerFiles arraylists, but it would make it slightly cleaner in the long run.